mmcv/runner/hooks/profiler.py

```diff
@@ -131,6 +132,12 @@ def _log_handler(prof):
                     raise ImportError('please run "pip install '
                                       'torch-tb-profiler" to install '
                                       'torch_tb_profiler')
+                if 'dir_name' not in trace_cfg:
+                    trace_cfg['dir_name'] = osp.join(runner.work_dir,
+                                                     'profile')
+                elif not osp.isabs(trace_cfg['dir_name']):
+                    trace_cfg['dir_name'] = osp.join(runner.work_dir,
+                                                     trace_cfg['dir_name'])
```

Comment:
Suggest adding `runner.logger.info(f'tracing files of ProfilerHook will be saved to {trace_cfg['dir_name']}.')`

Reply:
Done